server/data/restClient.ts

```diff
@@ -48,17 +41,25 @@ export default class RestClient {
     return this.config.timeout
   }
 
-  async get({ path = null, query = '', headers = {}, responseType = '', raw = false }: GetRequest): Promise<unknown> {
-    if (this.token) {
-      logger.info(`GET request using authentication token: calling ${this.name}: ${path} ${query}`)
+  private logRequest(method: string, path: string, query?: string) {
+    if (this.hmppsToken) {
+      logger.info(
+        `${method.toUpperCase()} request using HMPPS auth token: calling ${this.name}: ${path} ${query ?? ''}`
```

Comment:
Nice refactoring of all this logging stuff 👍 
Out of interest, what is the double question mark syntax? I've not seen that before:
```
`${method.toUpperCase()} request using HMPPS auth token: calling ${this.name}: ${path} ${query ?? ''}`
```

Reply:
It's just yet another syntax for a null check - if null then do that. Thinking about it I could maybe have used a parameter default instead. Might change that in another PR...